### PR TITLE
Fix internal error HTTP status code when exception reporting is on

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

### CODE REVIEW

File: thoth/user_api/openapi_server.py [hunk 1]
```diff
@@ -282,20 +282,23 @@ def internal_server_error(exc):
...
+                            "context": exc.__context__,
+                            "traceback": traceback.format_exception(exc_type, exc_value, exc_traceback),
+                            "args": list(exc.args),
+                        },
                     },
-                },
-            }
+                }
+            ),
+            500,
```
Comment (goern) [inline thoth/user_api/openapi_server.py:301]: nasty, nice catch... this should have a unit test some time...

:100:

**Comment by goern** [comment]
@fridex do you roll another release after this is merged?

**Comment by fridex** [comment]
> @fridex do you roll another release after this is merged?

It would be great to sync with Kevin on what is needed to address also https://github.com/thoth-station/thoth-application/issues/1200.

**Pull request merged**